src/mesh.cpp

```diff
@@ -2344,15 +2362,10 @@ void LibMesh::initialize()
   // assuming that unstructured meshes used in OpenMC are 3D
   n_dimension_ = 3;
 
-  m_ = make_unique<libMesh::Mesh>(*settings::libmesh_comm, n_dimension_);
-  m_->read(filename_);
-
   if (specified_length_multiplier_) {
```

Comment:
It looks like the `m_->prepare_for_use()` line still didn't recover its way back into `mesh.cpp` (probably something went wrong when we were rebasing yesterday). I believe that `m_->prepare_for_use()` needs to be called after `MeshTools::Modification::scale`. Would you please move the `if (specified_length_multiplier)...` part into the `initialize()` calls and then re-insert the `m_->prepare_for_use()` into the constructor for the file-based mesh?